src/features/Main/Footer/index.jsx

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+import './index.scss';
+
+export const Footer = () => (<footer><ul className="footer-links"><li><a href="https://pearsonadvance.com/privacy-notice/">Privacy Policy</a></li><li><a href="https://pearsonadvance.com/tos/">Terms of Service</a></li></ul></footer>);
```

Comment:
Line too long.

Reply:
@anfbermudezme if I try to seperate them in different lines I get a lint error.

`error  Unexpected block statement surrounding arrow body; move the returned value immediately after the =>`

Reply:
@anfbermudezme footer refactored!